### PR TITLE
Add Login/SignUp frontend

### PR DESCRIPTION
**Pull request by Michelle24211**
Code in client/components/Login

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import PostsListPage from './pages/PostsListPage';
 import PostFormPage from './pages/PostFormPage';
 import ShowPostPage from './pages/ShowPostPage';
 import AboutUsPage from './pages/AboutUsPage';
-
+import Login from './components/Login/Registration'
```
Comment (antonzabirko) [inline client/src/App.js:13]: Is there a benefit for having a subfolder for registration?

File: client/src/components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          </Modal>
+
+    </div>
+  )
+}
+
+//<label data-error="wrong" data-success="right" for="defaultForm-pass">Password</label>
+
+class Login extends Component{
+   state = {
```
Comment (antonzabirko) [inline client/src/components/Login/Login.js:36]: Perhaps for the login we don't want to store the password in memory

**Review by antonzabirko — COMMENTED** [review]
I think we should remove the state for login and use onFormSubmit instead.

File: client/src/App.js (hunk 1 above)
Comment (Michelle24211) [reply client/src/App.js:13]: The registration links the login and signup page, Login is the folder thats holds: Sign Up, Login , Registration

File: client/src/components/Login/Login.js (hunk 2 above)
Comment (Michelle24211) [reply client/src/components/Login/Login.js:36]: yep your right, it will change once edgardo post the user login

**Review by antonzabirko — APPROVED** [review]

File: client/src/components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import {Button, Modal} from 'react-bootstrap'
+
+function Log(props){
+  return(
+    <div>
+          <Modal id = "login" show = {props.show} onHide = {props.onRequestClose}>
```
Comment (DonatoM) [inline client/src/components/Login/Login.js:7]: 1. The indention looks inconsistent compared to the component above. 
2. You might want to also get rid of the extra spacing here. The same goes for the other components below.
    `<Modal id="login" show={props.show} onHide={props.onRequestClose}>`

File: client/src/components/Login/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          </Modal>
+
+    </div>
+  )
+}
+
+
+
+class SignUp extends Component{
+
```
Comment (DonatoM) [inline client/src/components/Login/SignUp.js:40]: Looks like you got some extra new lines here.

File: client/src/components/Login/Login.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import {Button, Modal} from 'react-bootstrap'
```
Comment (DonatoM) [inline client/src/components/Login/Login.js:2]: Missing an `;` at the end.

File: client/src/components/Login/Login.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+              <p>Not a Member?</p>
+             <Button className="btn-secondary" onClick = {props.onClick}>Sign Up</Button>
+            </Modal.Footer>
+          </Modal>
+
+    </div>
+  )
+}
+
+//<label data-error="wrong" data-success="right" for="defaultForm-pass">Password</label>
```
Comment (DonatoM) [inline client/src/components/Login/Login.js:33]: Might want to get rid of this.

File: client/src/components/Login/SignUp.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+  render() {
+    return (
+      <>
```
Comment (DonatoM) [inline client/src/components/Login/SignUp.js:44]: Do you need this?

**Pull request merged**